### PR TITLE
feat: Add ansys-tools-visualization-interface

### PR DESCRIPTION
**Pull request by AlejandroFernandezLuces**
Closes https://github.com/ansys/ansys-tools-visualization-interface/issues/38

### CODE REVIEW

File: doc/source/tools/libraries/index.rst [hunk 1]
```diff
@@ -10,7 +10,7 @@ Tooling libraries
       :text-align: center
       :class-title: pyansys-card-title
 
-      Simple gRPC API tool for moving files between a client and a remote server
+      Simple gRPC API tool for moving files between a client and a remote server.
```
Comment (RobPasMue) [inline doc/source/tools/libraries/index.rst:13]: Revert the points at the end of the phrases. We are avoiding them on purpose due to the way they render in cards

Comment (AlejandroFernandezLuces) [reply doc/source/tools/libraries/index.rst:13]: Some of them had them and some didn't, I'll remove all

Comment (RobPasMue) [reply doc/source/tools/libraries/index.rst]: Yeah the ones that had it, were probably my fault. Sorry.

File: doc/source/tools/libraries/index.rst [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Tooling libraries
 =================
 
-.. grid:: 3
+.. grid:: 2
```
Comment (RobPasMue) [inline doc/source/tools/libraries/index.rst:4]: Why this grid change¿?

File: doc/source/tools/libraries/index.rst [hunk 3]
```diff
@@ -50,7 +58,7 @@ Tooling libraries
       :text-align: center
       :class-title: pyansys-card-title
 
-      Tool for defining basic variables, types, metadata, and values intended to provide interoperability between all products.
+      Tool for defining basic variables, types, metadata, and values intended to provide interoperability between all products
 
     .. grid-item-card:: PyAnsys Tools Versioning
```
Comment (RobPasMue) [inline doc/source/tools/libraries/index.rst:63]: You should also add this info to the repo's main README. Info should be placed in both places

File: tools/links.py [hunk 4]
```diff
@@ -64,6 +64,7 @@
     "ansys-tools-local-product-launcher": "https://local-product-launcher.tools.docs.pyansys.com/version/stable",  # noqa: E501
     "ansys-tools-path": "https://path.tools.docs.pyansys.com/version/stable",
     "ansys-tools-protoc-helper": None,
+    "ansys-tools-visualization-interface": "https://visualization-interface.tools.docs.pyansys.com/",  # noqa: E501
```
Comment (RobPasMue) [inline tools/links.py:67]: You should also add the repo to the milestone.py script or workflow. I can't remember which one specifically

**Review by RobPasMue — CHANGES_REQUESTED** [review]
See my comments

File: doc/source/tools/libraries/index.rst (hunk 2 above)
Comment (RobPasMue) [reply doc/source/tools/libraries/index.rst:4]: All pages are using 3 - it's uniformity. Otherwise, every time we add a new one we would have to change it

Comment (AlejandroFernandezLuces) [reply doc/source/tools/libraries/index.rst:4]: So there was no row with one single element at the end. I agree with you, I'll revert

File: doc/source/tools/libraries/index.rst [hunk 5]
```diff
@@ -36,6 +36,14 @@ Tooling libraries
 
       Utility library to compile ``.proto`` files to Python source when building the package wheel
 
+    .. grid-item-card:: Ansys Tools Visualization interface
+      :img-top: ../../_static/thumbnails/intro.png
+      :link: https://visualization-interface.tools.docs.pyansys.com/
```
Comment (RobPasMue) [inline doc/source/tools/libraries/index.rst:41]:
```suggestion
      :link: https://visualization-interface.tools.docs.pyansys.com/version/stable
```
This part is important. The ``links.py`` file needs it AFAIR

**Review by RobPasMue — APPROVED** [review]
One minor suggestion - LGTM otherwise. Approving

**Comment by RobPasMue** [comment]
Good catch in https://github.com/ansys/pyansys/pull/589/commits/192b71f23f998ec314ec5b9defab14746e79596a @AlejandroFernandezLuces

**Pull request merged**